source/includes/reference/_users.md

```diff
@@ -62,7 +62,67 @@ details.address.city        | Address city name          | Text
 details.address.state  | Address state code State code. Required if country is US, CA, AU, BR. | Text
 details.address.occupation  | User occupation. Required for US, CA, JP         | Text
 
+## Get the currently logged in user
 
+> Example Request:
+
+```shell
+curl -X GET https://api.sandbox.transferwise.tech/v1/me \
+     -H "Authorization: Bearer <your api token>" 
+```
+
+> Example Response:
+
+```json
+{
+  "id": 294205,
+  "name": "Oliver Wilson",
+  "email": "oliver@gmail.com",
```

Comment:
I'd advise against using possibly existing emails, change to something like `oliver.wilson@example.com`